spec/controllers/address_controller_spec.rb

```diff
@@ -79,7 +77,8 @@
   end
 
   def current_app
-    @_current_app ||= SnapApplication.create!(
+    @_current_app ||= FactoryGirl.create(
```

Comment:
Hmm, do we not include the factory girl helper methods?

Feel like that's something we should/could do in the near future.